help/schedule-a-reminder.md

```diff
@@ -0,0 +1,58 @@
+# Schedule a reminder
+
+Zulip allows you to schedule a reminder for a message at a later time. The message is delivered
+to you as a [direct message](/help/direct-messages) by Notification Bot at the scheduled time.
+
+## Schedule a reminder
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!message-actions.md!}
+
+1. Click on the **ellipsis** (<i class="zulip-icon zulip-icon-more-vertical"></i>).
+
+1. Click **Remind me about this**.
+
+1. Select one of the suggested scheduling options, or pick a custom time.
+
+{end_tabs}
+
+## Delete a reminder
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!go-to-reminders.md!}
+
+1. Click the **delete** (<i class="zulip-icon zulip-icon-trash"></i>) icon on the message you
+   want to delete.
+
+!!! keyboard_tip ""
+
+    You can also use <kbd>Backspace</kbd> within the reminders view to
+    delete the selected reminder.
+
+{end_tabs}
+
+## View reminders
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!go-to-reminders.md!}
+
+{end_tabs}
+
+## Related articles
+
+
+* [Schedule a message](/help/schedule-a-message)
+* [Message drafts](/help/view-and-edit-your-message-drafts)
+* [Preview messages before sending](/help/preview-your-message-before-sending)
+* [Verify a message was sent](/help/verify-your-message-was-successfully-sent)
+* [Edit a message](/help/edit-a-message)
+* [Delete a message](/help/delete-a-message)
```

Comment:
```suggestion
* [Star a message](/help/star-a-message)
* [Marking messages as unread](/help/marking-messages-as-unread)
* [Schedule a message](/help/schedule-a-message)
```

Reply:
I think it's suggesting you remove some of the extra items in the "Related articles" section that had been copied from the "scheduled message" page that aren't as relevant.

Reply:
Thanks! Done

Reply:
And add other ones that are relevant, yes.